src/modules/visitor-space/components/FilterMenu/FilterMenuMobile/FilterMenuMobile.module.scss

```diff
@@ -88,6 +88,23 @@
 	height: calc(100% - #{$c-navigation-height-md});
 }
 
+.c-filter-menu-mobile__button:global(.c-button) {
+	position: relative;
+
+	&::before {
```

Comment:
Zou het overkill zijn om dit in een mixin oid te steken? Als dat er op mobile altijd hetzelfde uitziet als op desktop, dan moet dat maar op één plaats aangepast worden.

Reply:
Ik heb daar ook aan gedacht alleen was mijn grootste twijfel dat er in het geval van die filters, rekening gehouden. moet worden met een soort border van .1 rem. Dus als we dat generiek willen houden, zou die .1rem offset nog achteraf in rekening gebracht moeten worden. 

Langs de andere kant zou ik misschien wel een class kunnen maken en op beide plaatsen (want nu staat de code dubbel) dan @extend class gebruiken. Want denk jij?

Reply:
In dat geval, waar zetten we die util class dan 🤷‍♀️